src/metabase/automagic_dashboards/comparison.clj

```diff
@@ -149,25 +155,24 @@
   [card]
   (get-in card [:visualization_settings :graph.series_labels]
           (map (comp capitalize-first metric-name)
-               (qp.util/get-in-normalized card [:dataset_query :query :aggregation]))))
+               (get-in card [:dataset_query :query :aggregation]))))
 
 (defn- unroll-multiseries
   [card]
   (if (and (multiseries? card)
-           (-> card :display qp.util/normalize-token (= :line)))
+           (-> card :display mbql.u/normalize-token (= :line)))
```

Comment:
why does this need to still be normalized?

Reply:
Only the `dataset_query` is getting normalized, other properties of the Card still don't get normalized. `:display` is a `:keyword` Toucan type so we shouldn't *need* to normalize this (AFAIK the frontend always uses all-lower-case `"line"`, never `"LINE"`, but I wasn't trying to rewrite more code than I needed to. I broke enough things as it was. So I left this in here since it wouldn't hurt anything

Reply:
I will check with tom and see if this can be removed